pkg/agent/metrics/metrics.go

```diff
@@ -97,6 +101,15 @@ func newAgentMetrics() *AgentMetrics {
 		},
 		[]string{},
 	)
+	serverCount := prometheus.NewGaugeVec(
```

Comment:
I don't think there's any reason to use a vector metric if there aren't any labels? I could be wrong, but working with vector metrics is more expensive.

Reply:
I was just copying the previously existing metrics, which are all GaugeVecs without labels. What should I be using instead?

Reply:
You can use promethus.NewGauge() https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewGauge

Reply:
Done.